metadata-ingestion/tests/unit/test_kafka_source.py

```diff
@@ -26,106 +26,6 @@
 
 
 class KafkaSourceTest(unittest.TestCase):
-    def test_get_schema_str_replace_confluent_ref_avro(self):
```

Comment:
I removed the test because it depends on the confluent schema registry being part of the Kafka source. If your ok with the structure of the code i can refactor and move the test to `confluent_schema_registry` specific class